src/coreclr/jit/emitarm64.cpp

```diff
@@ -14758,21 +14779,25 @@ void emitter::emitIns_Call(EmitCallType          callType,
         case IF_SVE_AJ_3A:
         case IF_SVE_AL_3A:
         case IF_SVE_CL_3A:
-        case IF_SVE_DD_2A:
         case IF_SVE_DF_2A:
         case IF_SVE_GS_3A:
         case IF_SVE_HJ_3A:
         case IF_SVE_IY_4A:
             return PREDICATE_NONE;
 
+        case IF_SVE_DD_2A:
```

Comment:
This should be always `PREDICATE_NONE` for `DD_2A`, right? https://docsmirror.github.io/A64/2023-06/pfirst_p_p_p.html

Reply:
It needs `PREDICATE_NONE` for the second reg only else we will miss the element type on remaining predicate registers. It would cause the following diff with capstone.

```diff
- pfirst p0.b, p15, p0.b
+ pfirst p0, p15, p0

```

Reply:
@kunalspathak , this was one of the reasons why `insGetPredicateType` was originally called something like `insGetReg2PredicateType`.

Because this instruction has two predicate registers, `insGetPredicateType` is ambiguous of what it should return.

Reply:
I just now noticed the `regPos` was added. That seems to clear up any ambiguity.

Reply:
right...having `regPos` is better.